doc/CHEATSHEET.md

```diff
@@ -0,0 +1,145 @@
+# Cheatsheet
+
+## Run a trace
+
+* In a test
+    * `Tasty.testCase "foo" $ C.testSucceeds foo`
+    * `Tasty.testCase "foo" $ C.testFails foo`
+* In the REPL
+    * `printCooked $ runMockChain foo`
+
+## Use a custom initial distribution of value
+
+```haskell
+initDist :: InitialDistribution
+initDist = initialDistribution [(i, [lovelaceValueOf 25_000_000]) | i <- knownWallets]
+```
+* In a test `Tasty.testCase "foo" $ testSucceedsFrom def initDist foo`
+* In the REPL `printCooked $ runMockChainFrom initDist foo`
```

Comment:
Yes it was intended so that when you search for "run a trace" the given recipe is not polluted with anything else (does not use `From` alternatives) and when you search for "use a custom initial distribution" it shows what you need (here using the `From` alternatives).

I have no strong opinion about all these details, this is just a quick draft to illustrate what a cheatsheet may look like.